src/query/service/src/pipelines/processors/transforms/transform_async_function.rs

```diff
@@ -159,15 +160,6 @@ impl TransformAsyncFunction {
                         let remaining = max.saturating_sub(current);
                         let to_fetch = count.saturating_sub(remaining);
 
-                        let step_size = ctx.get_settings().get_sequence_step_size()?;
-                        let batch_size = to_fetch.max(step_size);
```

Comment:
We must have this line, if input has many rows, we should use max(rows, step_size) to increase the seq.